### PR TITLE
fix(internal): fix Model and add tests

### PR DESCRIPTION
**Pull request by sauyon**

### CODE REVIEW

**Comment by sauyon** [comment]
actually drafting this one because I need to rework yaml stuff.

**Comment by sauyon** [comment]
Will do that later.

File: bentoml/models.py [hunk 1]
```diff
@@ -82,8 +101,13 @@ def create(
...
 
-def load_runner(tag: t.Union[Tag, str]) -> "Runner":
-    model = get(tag)
+@inject
+def load_runner(
+    tag: t.Union[Tag, str],
+    *,
+    _model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+) -> "Runner":
+    model = get(tag, _model_store=_model_store)
```
Comment (parano) [inline bentoml/models.py:110]: cc @larme

File: tests/unit/_internal/models/test_model.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+from bentoml import Tag
+from bentoml.exceptions import BentoMLException
+from bentoml._internal.models.model import Model
+from bentoml._internal.models.model import ModelInfo
+from bentoml._internal.configuration import BENTOML_VERSION
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+expected_yaml = (
```
Comment (parano) [inline tests/unit/_internal/models/test_model.py:19]: would it be easier to write this with a multi-line string?

Comment (sauyon) [reply tests/unit/_internal/models/test_model.py:19]: probably, I wrote this originally as a bunch of f-strings before I realized that I needed to use format

File: tests/unit/_internal/models/test_model.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    with open(
+        sys_written_path, mode="w", encoding="utf-8", newline=""
+    ) as sys_written_file:
+        sys_written_file.write(sys_written_content)
+
+    with open(model_b.path_of("sys_written/file"), encoding="utf-8") as f:
+        assert f.read() == sys_written_content
+
+    b_tar_path = f"tar://{fs.path.join(str(tmpdir), 'modelb.tar')}"
+    model_b.export(b_tar_path)
```
Comment (parano) [inline tests/unit/_internal/models/test_model.py:115]: we we expect users to add `tar://` prefix when using import and export?

File: tests/unit/_internal/models/test_model.py [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    "  label: stringvalue\n"
+    "options:\n"
+    "  option:\n"
+    "    dictkey: dictvalue\n"
+    "metadata:\n"
+    "  metadatakey:\n"
+    "  - list\n"
+    "  - of\n"
+    "  - str\n"
+)
```
Comment (parano) [inline tests/unit/_internal/models/test_model.py:38]:
```suggestion
expected_yaml = """
name: test
version: version
bentoml_version: {bentoml_version}
creation_time: {creation_time}
api_version: v1
module: testmodule
context:
  ctx: 1
labels:
  label: stringvalue
options:
  option:
    dictkey: dictvalue
metadata:
  metadatakey:
  - list
  - of
  - str
"""
```

something like this?

**Review by parano — APPROVED** [review]

**Pull request merged**